README.md

```diff
@@ -1,208 +1,143 @@
-
-    // Update application state based on the response
-    let ssoToken, error;
-    if (step.type === 'LoginSuccess') {
-      ssoToken = step.getSessionToken();
-    } else if (step.type === 'LoginFailure') {
-      error = step.getCode();
-    }
-    this.setState({ step, ssoToken, error });
-  };
-
-  onError = (error) => {
-    this.setState({ error });
-  };
-
-  render() {
-    // Handle init/success/failure states
-    if (!this.state.step) {
-      return <p>Loading...</p>;
-    } else if (this.state.ssoToken) {
-      return <p>Authenticated!</p>;
-    } else if (this.state.error) {
-      return <p>Error: {this.state.error}</p>;
-    }
-
-    // Otherwise, select the correct component for this step
-    const stage = this.state.step.getStage();
-    return (
-      <>
-        <h1>Sign In</h1>
-        {stage === 'UsernamePassword' && (
-          <UsernamePassword step={this.state.step} onSubmit={this.nextStep} />
-        )}
-        {/* Create similar components for other stages */}
-      </>
-    );
-  }
-}
-
-// Custom UI for rendering the "UsernamePassword" step
-function UsernamePassword(props) {
-  const setUsername = (e) => {
-    const cb = props.step.getCallbackOfType('NameCallback');
-    cb.setName(e.target.value);
-  };
-
-  const setPassword = (e) => {
-    const cb = props.step.getCallbackOfType('PasswordCallback');
-    cb.setPassword(e.target.value);
-  };
-
-  const onSubmit = () => {
-    props.onSubmit(props.step);
-  };
-
-  return (
-    <>
-      <input type="text" placeholder="Username" onChange={setUsername} />
-      <input type="password" placeholder="Password" onChange={setPassword} />
-      <button onClick={onSubmit}>Submit</button>
-    </>
-  );
-}
-
-export default App;
+npm install @forgerock/javascript-sdk
 ```
 
-## Samples
+<!------------------------------------------------------------------------------------------------------------------------------------>
+<!-- QUICK START - Get one of the included samples up and running in as few steps as possible. -->
 
-Prerequisites:
+## Getting Started
 
-- [OpenSSL](https://www.openssl.org/) is installed
-- **samples/js/config.js** is updated to specify your SDK configuration
+To try out the ForgeRock JavaScript SDK, perform these steps:
 
-```bash
-# Add host
-echo '127.0.0.1 forgerock-sdk-samples.com' | sudo tee -a /etc/hosts
+1. Setup an Access Management (AM) instance, as described in the [documentation](https://sdks.forgerock.com/js/01_prepare-am/).
+2. Clone this repo:
 
-# Install dependencies
-npm i
+    ```
+    git clone https://github.com/ForgeRock/forgerock-javascript-sdk.git
+    ```
 
-# Generate CA and self-signed certificate
-# (Pick any passphrase and use it whenever prompted)
-npm run certs:make
+3. Open `samples/js/config.js` and edit the values to match your AM instance.
+4. Serve the `samples` directory by using a simple HTTP server, for example:
+   `npx http-server samples -c1 -p 3000`
```

Comment:
I should check this, but do we not need a secure connection (HTTPS) for this to work? Is running a local file considered secure?

Reply:
I'll check again locally, but I think localhost is considered one of the "secure origins" that Chromium is okay with. I'll clarify this in the text, and that it's for testing/eval purposes only.